src/filters/byte-format.js

```diff
@@ -0,0 +1,14 @@
+import { module } from 'angular';
+
+export default module('app.filters.byte-format', [])
+    .filter('byteFormat', () => (bytes, precision) => {
+      if (bytes === undefined) return '-';
+      if (bytes === 0) return '0 MB';
+      if (isNaN(parseFloat(bytes)) || !isFinite(bytes)) return '-';
```

Comment:
We use bytes as float later, so it isn't the worst idea to check if we can parse this as float